ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiMeteringReportService.scala

```diff
@@ -99,26 +103,54 @@ private[apiserver] object ApiMeteringReportService {
   class MeteringReportGenerator(participantId: Ref.ParticipantId) {
     def generate(
         request: GetMeteringReportRequest,
+        from: Timestamp,
+        to: Option[Timestamp],
+        applicationId: Option[Ref.ApplicationId],
         reportData: ReportData,
         generationTime: ProtoTimestamp,
     ): GetMeteringReportResponse = {
 
+      GetMeteringReportResponse(
+        request = Some(request),
+        participantReport = Some(genParticipantReport(reportData)),
+        reportGenerationTime = Some(generationTime),
+        meteringReportJson = Some(genMeteringReportJson(from, to, applicationId, reportData)),
+      )
+
+    }
+
+    // Note that this will be removed once downstream consumers no longer need it
```

Comment:
I'd have thought that Json report is the new standard, and the non-Json one will get eventually removed

Reply:
There still needs to be a `GetMeteringReportResponse` the `participantReport` will be removed the new `meteringReportJson` will live on.